pytest-pixl/src/pytest_pixl/keyvault.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright (c) 2022 University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""A mock Azure Keyvault instance for testing."""
+
+
+class MockKeyVault:
```

Comment:
feels like this should just belong in hasher api tests if we're only using it there? We can then remove pytest pixl from depdendencies

Reply:
Yeah was originally thinking of also using it for the `core.uploader` tests, which is why I put it here, but maybe not that useful.